library_generation/test/integration_tests.py

```diff
@@ -203,7 +203,7 @@ def __get_config_files(cls, path: str) -> List[tuple[str, str]]:
             if sub_dir.is_file():
                 continue
             repo = sub_dir.name
-            if repo == "golden":
+            if repo == "golden" or repo == "java-bigtable":
```

Comment:
Disable integration test in java-bigtable ([issue](https://github.com/googleapis/sdk-platform-java/issues/2549))